ci-operator/step-registry/ipi/install/install/in-bastion/ipi-install-install-in-bastion-commands.sh

```diff
@@ -271,6 +277,8 @@ echo "Installer exit with code $ret"
 # copy install artifacts from bastion host to ${dir} for reference
 echo "copy back installer artifacts from bastion host"
 run_scp_from_remote "${SSH_PRIV_KEY_PATH}" "${BASTION_SSH_USER}" "${BASTION_IP}" "${REMOTE_INSTALL_DIR}" "${dir%/*}"
+echo "[DEBUG] copy back squid access log from bastion host"
```

Comment:
do we still need this as we've already got step `gather-proxy` added in the `cucushift-installer-rehearse-gcp-ipi-custom-endpoints-proxy-whitelist-deprovision` chain

Reply:
Got it, just removed it. Thanks!